README.md

```diff
@@ -45,6 +50,23 @@ To upgrade:
 brew upgrade trenchman
 ```
 
+### Scoop installer (Windows, Intel and AMD)
+
+To install Trenchman via [Scoop](https://scoop.sh), run following commands:
+
+```console
+scoop bucket add scoop-clojure https://github.com/littleli/scoop-clojure
+scoop install trenchman
+```
+
+To upgrade use:
+
+```console
+scoop update *
```

Comment:
This line is doing more than just updating Trenchman, so I think I'd prefer to just `scoop update trenchman`.

Reply:
Ok. Makes sense. Let me fix this.